Ryujinx.Graphics.Gpu/Image/AutoDeleteCache.cs

```diff
@@ -104,6 +136,88 @@ public bool Remove(Texture texture, bool flush)
             return texture.DecrementReferenceCount();
         }
 
+        /// <summary>
+        /// Attempt to find a texture on the short duration cache.
+        /// </summary>
+        /// <param name="descriptor">The texture descriptor</param>
+        /// <returns>The texture if found, null otherwise</returns>
+        public Texture FindShortCache(in TextureDescriptor descriptor)
+        {
+            if (_shortCacheLookup.Count > 0)
```

Comment:
```suggestion
            if (_shortCacheLookup.Count > 0 && _shortCacheLookup.TryGetValue(descriptor, out var entry))
```
To avoid the nesting. Although it seems that the count > 0 check could be just removed, if the count is 0 then `TryGetValue` will not return true. Unless the count check is some kind of optimization.

Reply:
It's an optimisation to avoid any costs for the dictionary method, it looks like it does a bit more work than the count check and the large majority of the time it will be 0.